next/test/validation/json_schema_test_suite.test.ts

```diff
@@ -0,0 +1,51 @@
+import type { JsfSchema, SchemaValue } from '../../src/types'
+import fs from 'node:fs'
+import path from 'node:path'
+import util from 'node:util'
+import { describe, expect, it } from '@jest/globals'
+import { createHeadlessForm } from '../../src'
+
+interface Test {
+  description: string
+  data: SchemaValue
+  valid: boolean
+}
+
+interface TestSchema {
+  description: string
+  schema: JsfSchema
+  tests: Test[]
+}
+
+expect.extend({
```

Comment:
q: How do I run these tests? Obviously it says the test-suit direction does not exist, what am I missing 🤔  (And add it to the README instructions)

Reply:
Good point, because of the git submodule you will have to either
* run `git submodule update --init` in your existing repo
* or clone the repo with `git clone git@github.com:remoteoss/json-schema-form.git --recursive`
I will add that to the next Readme.

Reply:
I'll do a follow up PR for that.